tests/test_syslog.robot

```diff
@@ -0,0 +1,140 @@
+*** Settings ***
+
+Documentation       This suite is for testing syslog function of Open BMC.
+
+Resource            ../lib/rest_client.robot
+Resource            ../lib/utils.robot
+
+*** Variables ***
+${IP_ADDRESS}              9.109.116.67
```

Comment:
Existing syslog automation only checks if valid address is getting set.  We do not cross check if syslogs are getting logged in remote server. But I have tested it manually and it works. As suggested, I will try to enhance this in future.